src/NBitcoin/ScriptEvaluationContext.cs

```diff
@@ -1389,6 +1383,56 @@ private bool EvalScript(Script s, TransactionChecker checker, int hashversion)
                                         }
                                         break;
                                     }
+
+                                // Reads the federation id from the stack and then pushes the federation public keys 
+                                // to the stack. Typically preceeds an OP_CHECKMULTISIG operation.
+                                case OpcodeType.OP_FEDERATION:
+                                    {
+                                        if (this.Network.Federation == null)
```

Comment:
Later we can change this to check a ScriptVerify flag instead, but this will work initially

Reply:
Ok.